framework/decode/vulkan_ascii_consumer_base.cpp

```diff
@@ -39,28 +39,38 @@ VulkanAsciiConsumerBase::~VulkanAsciiConsumerBase()
 
 bool VulkanAsciiConsumerBase::Initialize(const std::string& filename)
```

Comment:
I suggest Initialize take a `FILE*`, then the management of the filename can be completely elevated to toascii main.

Reply:
I agree, but if it's not a major preference for you, I'd rather leave it as-is on this particular branch.